scripts/qa_crowbarsetup.sh

```diff
@@ -1451,6 +1451,7 @@ function reboot_nodes_via_ipmi
     [[ $ipmi_ip_addrs ]] || set_crowbar_ipmi_ip_addrs
     local ip
     local i=0
+    : ${want_ipmi_pxeextraopts:=options=persistent}
```

Comment:
We typically define defaults at the beginning of this script. Could you move this definition to the others?

Reply:
agreed, please move to the defaults section

but a nice idea indeed